### PR TITLE
math-magician:Components

### PR DESCRIPTION
**Pull request by PierreBarba**
# Hello reviewer, I hope you have a great day, thanks for the support any suggestions are welcome.
## JavaScript requirements

> Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

## Project requirements

> In the previous project, you worked on the setup of a React application using Create React App (CRA), now you should:

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
> In Calculator.js, create a React component that matches this design:
Calculator 

> Don't add any extra design.
> Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
> Use class based components (All components should be class-based).
> Once the Calculator is ready, import it and use it in your main component (App.js).
> Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          <button type="button" className="buttonCalculator">2</button>
+          <button type="button" className="buttonCalculator">3</button>
+          <button type="button" className="buttonCalculator color-right">+</button>
+          <button type="button" className="buttonCalculator zero">0</button>
+          <button type="button" className="buttonCalculator">.</button>
+          <button type="button" className="buttonCalculator color-right">=</button>
+        </div>
+      </section>
+    );
+  }
```
Comment (thecodechaser) [inline src/components/Calculator.js:36]: - [ ] You did a good job on the styles but you need to fix some issues. Please remove the white space between the output and input buttons, and also reduce the font size for the buttons.

### Your design

image

### Given design

image

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,72 @@
...
+  align-items: center;
+  justify-content: center;
+}
+
+.containerCalculator {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
```
Comment (thecodechaser) [inline src/index.css]: - [ ] On desktop screens the calculator is stretching and not looking good, Please make sure you fix this issue.
   - To fix this issue. You should give it a fixed width in `pixels` not in `%` ex: `width: 500px;`.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @PierreBarba👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 2 above)
Comment (PierreBarba) [reply src/index.css]: thanks for the thanks for the support any suggestions 😄

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @PierreBarba 

Your project is complete! Great job for making the required changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- your design looks ok
- calculator works fine
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**